internal/commands/secrets/flags.go

```diff
@@ -0,0 +1,11 @@
+package secrets
+
+const (
+	flagName      = "secret-name"
+	flagNameShort = "n"
+	flagNameUsage = `the name of the secret to add to your Realm App`
+
+	flagValue      = "secret-value"
+	flagValueShort = "v"
+	flagValueUsage = `the value of the secret to add to your Realm App`
```

Comment:
I know the spec has `-name` and `-value` as the shorthand here but since we can only support one character in the shorthand I used `n` and `v`